Src/GoogleApis.Auth.DotNet4/OAuth2/GoogleCredential.cs

```diff
@@ -100,7 +100,7 @@ public static GoogleCredential FromStream(Stream stream)
 
         /// <summary>
         /// <para>Returns <c>true</c> only if this credential type has no scopes by default and requires
-        /// a call to <see cref="CreateScoped"/> before use.</para>
+        /// a call to <see cref="o:CreateScoped"/> before use.</para>
```

Comment:
The "o:" prefix can be used to refer to all overloads. It's mentioned in [Sandcastle's docs](http://ewsoftware.github.io/XMLCommentsGuide/html/983fed56-321c-4daf-af16-e3169b28ffcd.htm#Syntax), though not in [MSDN](https://msdn.microsoft.com/en-us/library/fsbx0t7x.aspx).